apps/pkt-gen/pkt-gen.c

```diff
@@ -1570,6 +1690,13 @@ sender_body(void *data)
 			nexttime = timespec_add(nexttime, targ->g->tx_period);
 			wait_time(nexttime);
 		}
+		if (targ->garp) {
+			clock_gettime(CLOCK_REALTIME_PRECISE, &tmptime);
```

Comment:
Do we really need to be `CLOCK_REALTIME_PRECISE`? COARSE will be enough, and it's faster.

Reply:
Oh, the Linux manpage says COARSE is Linux-specific. I'm sure not what other supported platforms will need a work-around. How about gettimeofday?